templates/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+---
+{{ if .Values.ingress.create }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "sda.fullname" . }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
The option to add annotations is a must if utils like cert-manager is set up to request public certificates from LetsEncrypt.